services/docker-compose-ops.yml

```diff
@@ -69,6 +69,24 @@ services:
       timeout: 20s
       retries: 3
 
+  filestash:
+    container_name: filestash
+    image: machines/filestash:3a01b70
+    restart: always
+    ports:
+      - "9002:8334"
+    volumes:
+      - ${TMP_PATH_TO_FILESTASH_CONFIG}:/app/data/state/config/config.json
+    networks:
+      - simcore_default
+
+  onlyoffice:
```

Comment:
what is this?

Reply:
It's a service used to preview documents inside filestash. It also allows you to edit some types of files and save them back.